app/assets/stylesheets/nucore.scss

```diff
@@ -354,3 +354,12 @@ textarea.wide {
     display: inherit;
   }
 }
+
+/* Bootstrap adds an element with this class when you expand a dropdown and your
+ * browser supports the ontouchstart event. It claims to be necessary "if mobile
+ * we we use a backdrop because click events don't delegate". Instead, this
+ * is taking the tap and preventing the actual link to receive the click.
+ */
+.dropdown-backdrop {
+  display: none;
+}
```

Comment:
I also found this: https://stackoverflow.com/questions/17579750/drop-down-menu-not-working-on-mobile-devices

Initial testing, using `position: static;` seems to work as well. Both do seem to close the dropdown if you tap elsewhere on the page.